pontoon/messaging/templates/messaging/emails/monthly_activity_summary.html

```diff
@@ -0,0 +1,106 @@
+{% extends 'messaging/emails/base.html' %}
+
+{% block title %}{{ subject }}{% endblock %}
+
+{% block extend_css %}
+.subheader p:not(:last-child) {
+    padding-bottom: 10px;
+}
+
+h2 {
+    font-size: 18px;
+    font-weight: 100;
+    margin: 0;
+    padding-bottom: 10px;
+    text-transform: uppercase;
+}
+
+h2.team-activity {
+    padding-top: 30px;
+}
+
+p {
+    font-size: 15px;
+    line-height: 1.6em;
+    margin: 0;
+}
+
+p.completion {
+    padding-top: 20px;
+}
+
+.bullet {
+    display: inline-block;
+    border: 1px solid;
+    border-radius: 50%;
+    margin: 1px 4px;
+    height: 7px;
+    width: 7px;
+}
+
+.bullet.missing {
+    background: #bec7d1;
+    border-color: #bec7d1;
+}
+
+.bullet.unreviewed {
+    background: #4fc4f6;
+    border-color: #4fc4f6;
+}
+
+.bullet.pretranslated {
+    background: #c0ff00;
+    border-color: #c0ff00;
+}
+{% endblock %}
+
+{% block header %}Hello.{% endblock %}
+
+{% block subheader %}
+<p>Here's your monthly activity summary for the month of {{ month }}.</p>
+<p>A heartfelt thank you from the Mozilla Localization team and the countless users who benefit from your translations. Your dedication makes Mozilla’s products and services more accessible and enjoyable for people all over the world.</p>
```

Comment:
I thought something like this would be nice, saw something similar in the emails I'm getting from SUMO.